example-app/app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\City;
+use App\Models\Airline;
+
+class CityController extends Controller
+{
+    public function index(Request $request)
+    {
+        $query = City::withCount(['departureFlights', 'arrivalFlights'])
+            ->filter($request->input('airline_id'));
+
+        $query
+            ->when($request->has('sort'), function ($query) use ($request) {
+                $query->orderBy($request->sort, $request->order);
+            });
+
+        $cities = $query->simplePaginate(10);
+
+        if ($request->ajax()) {
+            return response()->json([
+                'table' => view('cities.partials.city_table', compact('cities'))->render(),
+                'pagination' => view('cities.partials.pagination', compact('cities'))->render()
+            ]);
+        }
```

Comment:
what about moving this to the `Api/CityController.php`

having that here breaks the S from SOLID: Single responsibility principle
https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design#single-responsibility-principle